modules/serverless-creating-broker.adoc

```diff
@@ -3,41 +3,144 @@
 // * serverless/knative_eventing/serverless-using-brokers.adoc
 
 [id="serverless-creating-broker_{context}"]
-= Creating a broker manually
+= Creating a Broker as a developer
 
-To create a broker, you must create a `ServiceAccount` for each namespace and give that `ServiceAccount` the required RBAC permissions.
+. Optional. If you do not have an existing event sink created to consume events from the Broker, you can create a simple Knative Service as an event sink.
+.. Copy the following into a YAML file:
++
 
-.Prerequisites
-* Knative Eventing installed, which includes the `ClusterRole`.
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display
+----
 
-.Procedure
-. Create the `ServiceAccount` objects:
+.. Create the Service by applying the YAML file:
 +
+
+[source,terminal]
 ----
-$ oc -n default create serviceaccount eventing-broker-ingress
-$ oc -n default create serviceaccount eventing-broker-filter
+$ oc apply -f <FILENAME>
 ----
 
-. Give those objects RBAC permissions:
+. Optional. Create an event source that emits events to the `default` Broker. The following example shows how you can create a basic PingSource.
+.. Copy the following into a YAML file:
 +
+
+[source,yml]
 ----
-$ oc -n default create rolebinding eventing-broker-ingress \
-  --clusterrole=eventing-broker-ingress \
-  --serviceaccount=default:eventing-broker-ingress
-$ oc -n default create rolebinding eventing-broker-filter \
-  --clusterrole=eventing-broker-filter \
-  --serviceaccount=default:eventing-broker-filter
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  name: ping-source
+spec:
+  schedule: "*/1 * * * *"
```

Comment:
Consider moving the commentary about the this schedule that you have in the verification step to here.

Reply:
I think it probably makes more sense where it is, since it's more of an explanation of what will be observed at the end than it is an explanation of how PingSource works.